tests/test_glmsingleunit.m

```diff
@@ -0,0 +1,50 @@
+function test_suite = test_glmsingleunit %#ok<*STOUT>
+
+  try % assignment of 'localfunctions' is necessary in Matlab >= 2016
+    test_functions = localfunctions(); %#ok<*NASGU>
+  catch % no problem; early Matlab versions can use initTestSuite fine
+  end
+
+  initTestSuite;
+
+end
+
+function test_glmsingleunit_smoke()
+
+  [data, output_dir] = set_up();
+
+  % GIVEN
+  design = data.design(1:3);
+  stimdur = data.stimdur;
+  tr = data.tr;
+  data = cellfun(@(x) x(51:70, 8:27, 1, :), data.data(1:3), 'UniformOutput', 0);
+
+  results = GLMestimatesingletrial(design, ...
+                                   data, ...
+                                   stimdur, ...
+                                   tr, ...
+                                   output_dir, ...
+                                   struct('wantmemoryoutputs', [1 1 1 1]));
```

Comment:
We could add an `assert` to make sure that the `results` values match what we "expect".

Reply:
Right.  I thought that was the main point of these types of tests.  (i.e., to see if any proposed changes changes the actual final answer.)